assets/css/screens/keywords/index.scss

```diff
@@ -0,0 +1,6 @@
+.keyword-index.table {
```

Comment:
For this screen name, we can implement the generic function to generate the body class name

```elixir
defmodule GoogleCrawlerWeb.LayoutView do
  use GoogleCrawlerWeb, :view

  def body_class(conn) do
    "#{module_class_name(conn)} #{action_name(conn)}"
  end

  defp module_class_name(conn) do
    controller_module(conn)
    |> Phoenix.Naming.resource_name("Controller")
    |> String.replace("_", "-")
  end
end
```
Add `action_name` and `controller_module` to `GoogleCrawlerWeb` - `view`

```elixir
import Phoenix.Controller,
        only: [get_flash: 1, get_flash: 2, view_module: 1, action_name: 1, controller_module: 1]
```

Then use it in the layout 👍 

```html
<body class="<%= body_class(@conn) %>">
```

Reply:
Thanks a lot @rosle , will add this